rascal-lsp/src/main/java/org/rascalmpl/vscode/lsp/rascal/RascalLanguageServices.java

```diff
@@ -281,6 +334,13 @@ public String getCommandName() {
         public String getShortName() {
             return shortName;
         }
+    }
 
+    public InterruptibleFuture<IList> codeActions(IList focus, PathConfig pcfg) {
+        return runEvaluator("Rascal codeActions", actionEvaluator, eval -> {
+            Map<String,IValue> kws = Map.of("pcfg", pcfg.asConstructor());
+            return (IList) eval.call("rascalCodeActions", "lang::rascal::lsp::Actions", kws, focus);
+        },
+        null, exec, false, client);
```

Comment:
Can we instead of `null`  return an empty list?

Reply:
check! Although this can never happen since `rascalCodeActions` returns the empty list by default.